src/services/createOrUpdateGoals.test.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable jest/no-disabled-tests */
 import faker from '@faker-js/faker';
+import { ResourceGroups } from 'aws-sdk';
```

Comment:
I don't think this is the correct import

Reply:
```suggestion
```